Library/Homebrew/cask/lib/hbc/artifact/binary.rb

```diff
@@ -9,7 +9,7 @@ def install_phase
 
       def link
         super
-        FileUtils.chmod "+x", source
+        @command.run!("/bin/chmod", args: ["+x", source])
```

Comment:
I've run `brew style --fix` and the indentation is fixed so I'll update the PR.

I'm not sure if `binary_spec.rb` needs to be updated.

Thank you for you help.